### ISSUE TITLE
[Sugestion] Return effects of cards

### ISSUE THREAD

**Issue by NiiMiyo** [AlanOC91/YGOPRODeck#276]
The API v7 accepts the `effect` param and only returns cards which have informed effect (Like "Draw", "Recover LP" or "Graveyard").
This is not stated on API Guide for v7.

I believe this information should also be returned with each card object as an array.

Also, I have noticed this does not work correctly. For example, searching for "Aroma" cards with effect "Recover LP" returns no cards, the website has the same problem.

Is this a beta feature?

**Comment by AlanOC91** [comment]
Yeah, this feature isn't shown on the API documentation as it's not currently 100% fit for public usage.
However, I will investigate the issues you have brought up and look into it!
